app/src/main/java/com/example/drawingactivity/DrawableAreaView.kt

```diff
@@ -0,0 +1,74 @@
+package com.example.drawingactivity
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.*
+import android.util.AttributeSet
+import android.util.Log
+import android.view.MotionEvent
+import android.view.View
+
+/**
+ * This view is the drawable area of the app.
+ * drawing on this location will draw to a bitmap which is loaded
+ * onBitmapDrawn saves the bitmap in the viewModel when invoked
+ * attemptBitmapUpdate will update the bitmap to be the same as the one in the viewModel when invoked
```

Comment:
Don't ever assume someone is using this with a viewmodel

Reply:
Updated to be a more general overview of what the view can do, rather than how it is used in this context.